dependencies.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import sys
+import subprocess
+import json
+
+# Set default logging level to INFO
+logging.basicConfig(level="INFO")
+
+# Check if current python version is 3.8
+if sys.version_info < (3, 8):
+    logging.fatal('McPy needs Python version 3.8.0 or higher to run! Current version is %s.%s.%s' % (
+        sys.version_info.major, sys.version_info.minor, sys.version_info.micro))
+    sys.exit(-4)
+
+# If the OS is Linux, the module "distro" will import
+try:
+    import distro
+except ModuleNotFoundError: # Windows and MacOS will not find this module, because they don't have distributions
+    pass
+
+# Make sure pip is installed
+logging.info("Making sure pip is installed...")
+try: # Debian and Ubuntu require pip to be installed differently because they disable ensurepip
+    linuxDistro = distro.linux_distribution()
+    if linuxDistro[0] == "Debian":
+        subprocess.check_call(['sudo', 'apt', 'install', 'python3-pip'])
+    if linuxDistro[0] == "Ubuntu":
```

Comment:
1) Can be merged
2) This will likely also fail as if the application is not run under the superuser (if that isn't the case we have a security issue)
I would change this into a warning/fatal message or at least in some way check if the application is currently under superuser

Reply:
Now catches exception to log a fatal error if ran without superuser on Linux

Reply:
It can still be merged into something like
```
if linuxDistro[0] == "Debian" or linuxDistro[0] == "Ubuntu":
```
or similar